tests/ui/statics/unsized_type2.rs

```diff
@@ -0,0 +1,20 @@
+//! This test used to actually start evaluating the static even though
+//! there were errors in typeck.
```

Comment:
Could you link the issue?

```suggestion
//! there were errors in typeck.
//! issue: rust-lang/rust#123153
```